src/app/utils/python/chunkText.py

```diff
@@ -0,0 +1,52 @@
+import json
+from typing import List, Dict
+
+
+def chunkString(string: str, window_size: int, window_jump: int) -> List[str]:
+    start_pos = 0
+    end_pos = start_pos + window_size
+    chunks = []
+    while end_pos < len(string):
+        chunk = string[start_pos:end_pos]
+        chunks.append(chunk)
+        start_pos += window_jump
+        end_pos = start_pos + window_size
+    if end_pos > len(string):
+        end_chunk = string[start_pos:-1]
+        chunks.append(end_chunk)
+    return chunks
+
+
+def chunkText(scraped_pages: List[Dict[str, str]]) -> List[str]:
```

Comment:
Does `Dict[str, str]` state that the key is a `str` and the value is also `str` or that the dict is of size 2?

Reply:
This means that the key and value are both of type str